thunder/core/jit_ext.py

```diff
@@ -617,6 +618,25 @@ def _general_jit_torch_finfo_lookaside(dtype: thunder.dtypes.dtype):
     return res
 
 
+@register_general_jit_lookaside(torch.utils.checkpoint.checkpoint)
+def _general_jit_torch_checkpoint_lookaside(
```

Comment:
Would you add a comment / docstring explaining this function and its intended use?

Reply:
Yes, will do!